examples/flower_example.rs

```diff
@@ -0,0 +1,35 @@
+use polars::frame::DataFrame;
+use polars::series::Series;
+use polars::error::PolarsResult;
+use polars_core::prelude::*;
+use polars::prelude::NamedFrom;
```

Comment:
Usually the prelude is meant to be entirely included (with `use polars::prelude::*`).   Any reason not to do so for this one?

Reply:
I was just following an example